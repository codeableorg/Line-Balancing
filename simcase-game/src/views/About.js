/** @jsx jsx */
import React from "react";
import { jsx } from "@emotion/core";
import { Link } from "@reach/router";

import Navbar from "../components/Navbar";
import { Button } from "../components/ui";
import { MainContent, Action } from "../components/helpers";

const container = {
  h2: {
    textAlign: "center"
  },
  section: {
    display: "flex",
    flexDirection: "column",
    justifyContent: "space-between",
    alignItems: "center"
  },
  img: {
    width: "300px"
  },
  p: {
    margin: "5px"
  }
};

const button = {
  width: "50%"
};

function About() {
  return (
    <>
      <Navbar />
      <MainContent styles={container}>
        <h2>About</h2>
        <section>
          <figure>
            <img src="/assets/img/simcase_color_big.png" alt="Simcase logo" />
          </figure>
          <p css={{ textAlign: "center" }}>Developers in charge</p>
          <section>
            <p>
              <span role="img" aria-label="developer">
                üê∑
              </span>{" "}
              Cesar Cachay
            </p>
            <p>
              <span role="img" aria-label="developer">
                ü¶ç
              </span>{" "}
              Deyvi Conde
            </p>
            <p>
              <span role="img" aria-label="developer">
                ü¶ù
              </span>{" "}
              Paulo Tijero
            </p>
            <p>
              <span role="img" aria-label="developer">
                üëΩ
              </span>{" "}
              Carlos Ayala
            </p>
          </section>
        </section>
      </MainContent>
      <Action>
        <Button styles={button}>
          <Link to="/game/1">Play Again</Link>
        </Button>
        <Button styles={button}>
          <Link to="/">Go Home</Link>
        </Button>
      </Action>
    </>
  );
}

export default About;
