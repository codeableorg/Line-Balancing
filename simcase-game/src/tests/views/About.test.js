import React from "react";
import { render } from "@testing-library/react";

import About from "../../views/About";

test("render about", () => {
  const { asFragment } = render(<About />);
  expect(asFragment()).toMatchSnapshot();
});
