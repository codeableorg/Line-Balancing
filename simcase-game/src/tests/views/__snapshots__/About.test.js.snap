// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render about 1`] = `
<DocumentFragment>
  Hello from About Us View
  <a
    aria-current="page"
    href="/"
  >
    Go Back
  </a>
</DocumentFragment>
`;
